Examples/AblyCocoaAPNSExample/README.md

```diff
@@ -0,0 +1,10 @@
+This app gives you a brief understanding of how Ably's Push Notifications work. 
+You will need a real iOS device to test this functionality.
+
+- First, go to https://ably.com/accounts/, create an app and copy API key from API Keys tab of the app's dashboard.
+- Insert it instead of an empty string in the `key` property's value inside the `AblyHelper` class.
+- Make sure you've selected your development team in the `Signing & Capabilities` tab of the Xcode project target settings and all the provisioning profiles created without errors.
+- Build and Run the app on your device.
+- Hit "Activate" button, then hit "Print Token". You should see the printed value in the debug output window. You can also hit "Device Details" for additional check.
```

Comment:
What is the idea of this step? And what is the "additional check"? Here's a screenshot of what I see when I press "Device Details" - I'm not sure what I might want to check there.

![IMG_4920](https://user-images.githubusercontent.com/53756884/161753554-94b7ea41-1160-4978-8e5a-1d871b79e331.PNG)

Reply:
"additional check" I've meant whether device was successfully activated

Reply:
I think that it needs to make sense to someone who's seeing Ably / our push notifications stuff for the first time, I don't think it's clear as it is.

Reply:
Naming https://github.com/ably/ably-cocoa/pull/1319/commits/f310e5cfde939cf37df42b29d4753a52fd8b83d6
"additional check" comment https://github.com/ably/ably-cocoa/pull/1319/commits/edbda42f93c604d0bd5cfb1a516f73168b1695e8